recipe/conda_build_config.yaml

```diff
@@ -117,16 +121,18 @@ libmatio:
   - 1.5
 libnetcdf:
   - 4.4
+libpcap:
+  - 1.8
 libpng:
-  - 1.6
+  - 1.6.32
```

Comment:
There was a [rather strange constraint]( https://github.com/conda-forge/conda-forge.github.io/blob/61cdd7fd0374c07e95afdcbd414c7985b141e4b3/scripts/pin_the_slow_way.py#L95 ). Probably mostly based on our desire to use the same pinnings in `build` and `run`. This was the lower end of that constraint. So think this should be ok going forward, but we may need to take the high end of the constraint ( `1.6.34` ) if we want to be really conservative.